test/integration/reset.cc

```diff
@@ -17,6 +17,9 @@
 
 #include <gtest/gtest.h>
 
+#include <gz/msgs/boolean.pb.h>
+#include <gz/msgs/world_control.pb.h>
```

Comment:
this comment is actually for `test/integration/reset_sensors.cc`:

We'll also need to add `#include <gz/msgs/world_control.pb.h>` to that file to fix a compile error